### ISSUE TITLE
Noncirculating Holding Locations

### ISSUE THREAD

**Issue by tampakis** [pulibrary/locations#48]
It seems like the "always_requestable" column is something distinct from actual non-circulating locations (though it seems like all of the always requestable locations are non-circulating). Add a `circulating` column to also include reference locations and the following Firestone locations (thanks Joan!):
```
clasnc
dr
dss
egsr
fis
fnc
hldn 
necnc
sc
sd
se
sh
shp
shs
slav
sne
sp
spc
spl
sps
srel
ssa
sss
xlnc
```

**Comment by kevinreiss** [comment]
Do you mean a ```non_circulating``` column? The always_requestable flag was put in with the intent of showing items at a location as being "Available" regardless of their item status, and to display a "special message" to directing an interested user to check with the circulation desk associated with the location about items in these locations.

**Comment by tampakis** [comment]
Either `non_circulating` with all of these codes set to true or `circulating` with all of these codes set to false. I wanted there to be a single column to determine the "On-Site Access" status.

**Comment by kevinreiss** [comment]
Sounds good @tampakis. ```circulating``` is actually a better column name since it will be the default behavior at most locations.

**Comment by tampakis** [comment]
Closed by #49.
